src/common/hf.ts

```diff
@@ -108,30 +104,37 @@ export const hfValues = async (
     const schema = await getValuesSchema()
     const allSecrets = extract(schema, 'x-secret')
     const allSecretsPaths = Object.keys(flattenObject(allSecrets))
-    allSecretsPaths.forEach((path) => {
-      if (has(res, path)) set(res, path, '<redacted>')
+    allSecretsPaths.forEach((filePath) => {
+      if (has(res, filePath)) set(res, filePath, '<redacted>')
     })
   }
 
   if (withWorkloadValues) {
-    const files = {}
-    const filePaths = await glob([
-      `${envDir}/env/teams/workloads/**/*.yaml`,
-      `${envDir}/env/teams/*/sealedsecrets/*.yaml`,
-    ])
-
-    await Promise.allSettled(
-      filePaths.map(async (path) => {
-        const relativePath = path.replace(`${envDir}/`, '')
-        files[relativePath] = (await readFile(path)).toString()
-      }),
-    )
+    const files = await getStandaloneFiles(envDir)
     res.files = files
   }
 
   return res
 }
 
+// Get file content for those files that are not automatically loaded to the spec
+export const getStandaloneFiles = async (envDir: string): Promise<Record<string, any>> => {
+  const files = {}
+  const maps = getFileMaps(envDir).filter((map) => map.loadToSpec === false)
+  const pathGlobs = maps.map((fileMap) => {
+    return fileMap.pathGlob
+  })
+  const filePaths = await glob(pathGlobs)
+
+  await Promise.allSettled(
+    filePaths.map(async (path) => {
+      const relativePath = path.replace(`${envDir}/`, '')
+      files[relativePath] = (await readFile(path)).toString()
```

Comment:
I think its better to specify the encoding right?
```suggestion
      files[relativePath] = await readFile(path, 'utf8')
```
Also I always try to avoid mutating an object when you're in a promise, it can lead to weird race conditions. I think its better to just return an immutable object from the promise.